hack/create.sh

```diff
@@ -75,21 +75,7 @@ properties:
 EOF
 
 go generate ./...
-go run cmd/createorupdate/createorupdate.go -loglevel=debug
-
-az group deployment create -g $RESOURCEGROUP -n azuredeploy --template-file _data/_out/azuredeploy.json --no-wait
-
-if [[ "$RUN_SYNC_LOCAL" == "true" ]]; then
-    # will eventually run as an HCP pod, for development run it locally
-    # sleep until FQDN/healthz responds with OK. It takes up to 5 min for API server to respond. 
-    FQDN=$(awk '/^  fqdn:/ { print $2 }' <_data/manifest.yaml)
-    while [[ "$(curl -s -k https://${FQDN}/healthz/ready)" != "ok" ]]; do sleep 5; done
-    KUBECONFIG=_data/_out/admin.kubeconfig go run cmd/sync/sync.go -run-once=true -loglevel=debug
-fi
-
-az group deployment wait -g $RESOURCEGROUP -n azuredeploy --created --interval 10
-
-KUBECONFIG=_data/_out/admin.kubeconfig go run cmd/healthcheck/healthcheck.go -loglevel=debug
+go run cmd/createorupdate/*.go -loglevel=debug
```

Comment:
Why * is needed?

Reply:
because I shamelessly added multiple .go files under cmd/ instead of adding a separate package, generally because I think this will be further refactored after discussing with MSFT (I wouldn't be surprised if we end up using our upgrade orchestration).

Reply:
> Elaboration would be nice here :)

The RP uses our upgrade code